Cargo.toml

```diff
@@ -13,27 +13,27 @@ default = []
 sdl = ["gfx_window_sdl"]
 vulkan = ["gfx_device_vulkan", "gfx_device_vulkanll", "gfx_window_vulkan"]
 metal = ["gfx_device_metal", "gfx_window_metal"]
+serialize = ["gfx_core/serialize"]
 unstable = []
 
-
 [lib]
 name = "gfx_app"
 
 [dependencies]
 log = "0.3"
-env_logger = "0.3"
-glutin = "0.7.1"
-winit = "0.5.1"
-gfx_core = { path = "src/core", version = "0.6" }
+env_logger = "0.4"
+glutin = "0.7"
```

Comment:
I think you are misreading the spec slightly. For 0.x.y, AFAIK, `x` is considered a "major" and `y` as "minor". Adding features just requires you to bump the patch version then, as long as it's backwards compatible. If something **depends** on new features, it needs to have the patch version specified for that minimal version (what we have). `cargo update` would still use the newer patch versions, if available, but it will guarantee we'll not get lower ones.